### PR TITLE
add run-kubectl workflow

### PR DESCRIPTION
**Pull request by martinhny**

### CODE REVIEW

File: .github/workflows/run-kubectl.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      - id: 'get-credentials'
+        uses: google-github-actions/get-gke-credentials@v2
+        with:
+          cluster_name: ${{ env.CLUSTER_NAME }}
+          location: 'europe-north1'
+          use_connect_gateway: 'true'
+          project_id: ${{ env.KUBERNETES_PROJECT_ID }}
+          namespace: ${{ env.NAMESPACE }}
+
+      - name: 'Deploy the app'
```
Comment (evenh) [inline .github/workflows/run-kubectl.yaml:68]:
```suggestion
      - name: 'Execute kubectl'
```

**Review by evenh — APPROVED** [review]
Open question: what version of kubectl is used? Should we make it configurable with a modern/sane default?

File: .github/workflows/run-kubectl.yaml (hunk 1 above)
Comment (martinhny) [reply .github/workflows/run-kubectl.yaml]: fixed

**Comment by martinhny** [comment]
> Open question: what version of kubectl is used? Should we make it configurable with a modern/sane default?

good question. I added the https://github.com/Azure/setup-kubectl/tree/main action to give us more control, previously it was just using whatever came bundled with ubuntu. Now it will use the latest stable or whichever we configure.

**Review by evenh — APPROVED** [review]

**Review by BardOve — APPROVED** [review]

**Pull request merged**